web/src/lib/data-hooks/useUserData.ts

```diff
@@ -115,7 +115,10 @@ export const useUserData = (): UseUserDataResponse => {
   };
 
   const refresh = async (): Promise<void> => {
-    await mutate();
+    const updatedUserData = await mutate();
+    if (updatedUserData) {
+      setUpdateQueue(new UpdateQueueFactory(updatedUserData, update));
```

Comment:
This is necessary because it was an oversight from the recent change to how the update queue is used in the TaskHeader and Checkbox components. 

Without this line, the updateQueue does not updated with new data when the license status refresh occurs